swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/execute-tests-in-parallel.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: ballerina-testing-code-left-nav-pages-swanlake
+title: Execute tests in parallel
+description: Learn how to execute tests in parallel.
+keywords: ballerina, programming language, testing, test execution
+permalink: /learn/test-ballerina-code/execute-tests-in-parallel/
+active: execute-tests
+intro: The sections below include information about executing tests in parallel.
+---
+
+## Enable parallel execution of tests
+
+Tests are executed serially by default. User has to consciously enable the parallel execution by passing
```

Comment:
```suggestion
Tests are executed serially by default. The user has to explicitly activate the parallel execution by passing
```

Reply:
instead of `by passing` , should we use `by providing`?